src/sentry/api/serializers/models/commit.py

```diff
@@ -68,14 +69,17 @@ def serialize(self, obj, attrs, user):
             d['repository'] = attrs['repository']
         if 'author' not in self.exclude:
             d['author'] = attrs['user']
+        if 'pk' in self.include:
+            d['pk'] = obj.id
```

Comment:
Should we be exposing this as `id`? I don't think we use `pk` elsewhere in the API.

Reply:
It's kind of a pain, the suspect commits code mixes the logic of actually determining the suspect commits with serializing them. Maybe I'll take the time to refactor that code so that I don't need to do this.